### DISCUSSION TITLE
Is it possible to get different routes / views for the same model?

### DISCUSSION THREAD

**Discussion by raphael-allard** [mikker/passwordless#212] [Q&A]
I have a Rails application that uses the Passwordless gem to allow our partner stores to log in: several of their employees log in to the same session using the unique email address of the store. It's perfect because it prevents them from logging in from home!

I'm currently redesigning this space, and while I'm working on making it fully functional, I'd like to keep the old one accessible. For the redesign, I've created an engine within my app to separate it from the significant structure of our Webapp (we have several distinct login spaces). The issue is that I'd like to have new login routes along with a view tailored to this new space. Is it possible to set this up?

What surprises me is that I don't have a suffix in front of my current routes:

```
# config/routes.rb
Rails.application.routes.draw do
  passwordless_for :point_of_sale_users
end

$ rails routes | grep passwordless
      sign_in GET        /sign_in(.:format)        passwordless/sessions#new
              POST       /sign_in(.:format)        passwordless/sessions#create
token_sign_in GET        /sign_in/:token(.:format) passwordless/sessions#show
     sign_out GET|DELETE /sign_out(.:format)       passwordless/sessions#destroy
```

I hope I'm clear enough! Thanks for all the work, by the way; it's a great gem!

**Comment by mikker** [comment]
1. You could build a custom setup. Copy whatever you need from the included controller as a starting point.
2. Look into the `as:` and `at:` options for `passwordless_for`.

**Reply by raphael-allard** [reply]
Thank you for your prompt reply, @mikker!

I attempted to customize my routes by adding the `as:` and `at:` options in the `passwordless_for` method:

```
passwordless_for :point_of_sale_users, at: 'session_stuff', as: :user_session_things
```

However, when executing `$ rails routes | grep passwordless`, I still receive the same routes as in my previous message. I've checked the file `lib/passwordless/router_helpers.rb` from version 0.12.0 (the current version installed in my app), but the options seem to work quite similarly to the latest version.

Any ideas what could provoke this issue? Thanks
